src/pages/docs/canvas/getting-started/variables.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 'Variables'
+---
+
+Blutui passes variables to the templates for manipulation in the template. Variables may have attributes or elements you can access, too.
+
+Use a dot (`.`) to access attributes of a variable:
+
+```canvas {% process=false %}
+{{ foo.bar }}
+```
+
+It’s important to know that the curly braces are not part of the variable but the print statement. When accessing variables inside tags, don’t put the braces around them.
+
+If a variable or attribute does not exist, you will receive a **null** value.
+
+If you want to access a dynamic attribute of a variable, use the [attribute](/docs/canvas/functions/attribute) function instead. The **attribute** function is also useful when the attribute contains special characters (like **-** that would be interpreted as the minus operator):
+
+```canvas {% process=false %}
+{# equivalent to the non-working foo.data-foo #}
+{{ attribute(foo, 'data-foo') }}
+```
+
```

Comment:
I see that the old docs have global variables, or has this been removed in blutui so not needed in docs

Reply:
I removed that section because I felt that those default global variables added no value. I feel that users are more likely to use the Blutui's custom global variables instead of the defaults, eg. `request`, `route`, `settings`, etc.